phaseportrait/PhasePortrait3D.py

```diff
@@ -21,6 +21,41 @@ class PhasePortrait3D:
     -------
     ![image](../../imgs/doc_examples/pp3d_example.png)
 
+    ```python3
```

Comment:
Does your editor does highlighting with python3? Mine does not. But it works with `py` or `python`. Could you change it to `py` so it's like PhasePotrait2D example?